tests/model_serving/model_server/components/kserve_dsc_deployment_mode/test_kserve_dsc_default_deployment_mode.py

```diff
@@ -170,3 +171,39 @@ def test_restarted_pod_is_raw(
         pod_containers = {container.name for container in restarted_inference_pod.instance.spec.containers}
 
         assert pod_containers == {"kserve-container"}
+
+    def test_default_service_config_is_headless(
+        self, admin_client, default_deployment_mode_in_dsc, ovms_inference_service
+    ):
+        """
+        Verify that the default service configuration is set to Headless mode
+        (serviceClusterIPNone=True).
+        """
+        assert get_service_cluster_ip(admin_client) is True, (
+            "Default service config should be Headless (serviceClusterIPNone=True)"
+        )
```

Comment:
1. please use arg=val
2. please save the reutnr value of `get_service_cluster_ip` and log in the error
3. please use `if..... pyteat.fail....`